utils/build.ps1

```diff
@@ -2052,6 +2052,7 @@ if ($Test -contains "swiftpm") { Test-PackageManager $HostArch }
       $Indent += 2
   }
 
+  Write-ErrorLines "env:Path $env:Path"
```

Comment:
Does it make sense to be _after_ the error message? Perhaps hoist this before the error message?